### ISSUE TITLE
FlowNet2C training and validation

### ISSUE THREAD

**Issue by Queenyy** [ClementPinard/Pytorch-Correlation-extension#46]
@ClementPinard Hi, thanks for your great work, I used it when training FlowNet2C. Install is well-done. I use `                SpatialCorrelationSampler(kernel_size=1, patch_size=21, stride=1, padding=0, dilation=1, dilation_patch=2)
` to replace the correlation function in the code. But the training loss didn't decrease and the validation EPE is very big. I don't know why, could you please give me some suggestions? Thanks very much. 
![image](https://user-images.githubusercontent.com/29726357/90636123-bd5bcc80-e25c-11ea-994a-de3ffb1d70bf.png)

**Comment by ClementPinard** [comment]
Hi, thanks for your interest in this repo.
Can you add some details ? What code do you use exactly ? Do you have a repo name ? I used this repo for FlowNetC in my own FlowNetPytorch repo without problem.

**Comment by Queenyy** [comment]
@ClementPinard Thanks for your quick reply. I use the NVIDIA/flownet2-pytorch[https://github.com/NVIDIA/flownet2-pytorch](url) repo. When i use its own correlation function to train FlowNet2C, the loss is bumping, but train FlowNet2S can get a right result. So I guess the bad result on FlowNet2C is because of correlation layer, thus i used your work to replace the original function. It seems that the loss is still bumping. 
I use cuda 10.0, pytorch 1.2, gcc 4.4.7. is there some problems here?
Do you have other trick to train FlowNet2C?  
Extremely Grateful.
